scripts/generator/graphql_generator/csharp/SDK/ShopifyError.cs.erb

```diff
@@ -16,7 +16,7 @@ namespace <%= namespace %>.SDK {
         }
 
         public enum ErrorType {
-            HTTP, GraphQL, UserError, NativePaymentProcessingError
+            HTTP, GraphQL, UserError, NativePaymentProcessingError, WebCheckoutVerificationError
```

Comment:
Yes this might be better, they both technically are validating that the checkout is valid. We would have to rename `NativePaymentProcessingError` though